wiki/People/Global_Moderation_Team/en.md

```diff
@@ -74,6 +74,7 @@ The [Global Moderation Team group page](https://osu.ppy.sh/groups/4) lists all o
 
 | Name | Additional languages | Area of focus |
 | :-- | :-- | :-- |
+| ::{ flag=MY }:: [_Kobii](https://osu.ppy.sh/users/6209713) |  | Chat moderation, mapping/modding community moderation |
```

Comment:
do any of the gmt additions here require edits to the "by languages moderated" table below?

Reply:
they were all mapmod gmts, so i'd assume no, that was not their primary area of focus. i've updated the files to reflect that. i especially remember spectator didn't want to be a chat moderator anyway

Reply:
there’s an important distinction between moderating the language and being fluent in it, specifically the moderated channels list at the bottom. i think the languages should be included so that anyone with questions can be directed to them, even if they don’t actively check the relevant chats/forums.

Reply:
did you mean like adding the languages spoken, but not adding them to the Grouped by languages moderated list?

Reply:
i'll update these once _kobii and shad0wstar have replied with what languages they speak besides english, if any

Reply:
> did you mean like adding the languages spoken, but not adding them to the Grouped by languages moderated list?

I didn't even notice originally that they were missing entries in languages spoken, but yes that would be good.